cl/assets/templates/cotton/data_table.html

```diff
@@ -1,16 +1,24 @@
 {% load extras %}
 
-<c-vars columns rows footer caption="Data table" safe></c-vars>
+<c-vars columns rows footer align caption="Data table" safe></c-vars>
 
-<div class="w-full overflow-auto">
-  <table>
+<div class="overflow-auto">
+  <table class="w-full">
     <caption class="sr-only">{{ caption }}</caption>
     <thead>
     <tr class="border-none">
       {% for column in columns %}
         <th
-          class="text-nowrap text-start text-sm p-2 bg-greyscale-100 text-greyscale-700 font-normal {% if forloop.first %}rounded-l-lg{% elif forloop.last  %}rounded-r-lg{% endif %}"
-        >
+          class="text-nowrap text-sm p-2 bg-greyscale-100 text-greyscale-700 font-normal
+            {% if forloop.first %}rounded-l-lg{% elif forloop.last  %}rounded-r-lg{% endif %}
+            {% if align == "data-center" and not forloop.first %}
+              text-center
```

Comment:
These are consistent with the previous code for applying rounded classes, but I've gotten a bit more familiar with Tailwind since then, and it turns out there are quite a few handy [modifiers][tw-docs] (first, last, odd, etc.) we can use to simplify this quite a bit:

```suggestion
            first:rounded-l-lg last:rounded-r-lg
            {% if align == "data-center" %}
              text-center first:text-start
```

This feels a little easier to read, and it stays aligned with our goal of sticking to Tailwind for styling. What do you think?

[tw-docs]: https://v3.tailwindcss.com/docs/hover-focus-and-other-states#pseudo-class-reference